### PR TITLE
GEDI L3 VEDA Guidelines

### PR DESCRIPTION
**Pull request by smk0033**
Reference ticket: https://github.com/orgs/NASA-IMPACT/projects/29?pane=issue&itemId=30816576

Updated GEDI_L3 per new guidelines, added new sections, readjusted "About the Data"

### CODE REVIEW

**Review by jjfrench — APPROVED** [review]
Changes look good to me

**Comment by smk0033** [comment]
Thanks!! Will wait until this afternoon to merge in case anyone else has any feedback

**Review by omshinde — COMMENTED** [review]
Thanks @smk0033 ..Changes look good to me as well

**Pull request merged**
